update: actions.rst typo (#4302)
OttoWinter/esphomedocs: automations/actions.rst

``if`` Action
*************

> This action first evaluates a certain condition (``if:``) and then either

Reviewer: ```suggestion
This action first evaluates the ``condition:`` and then either
```

> executes the ``then:`` branch or the ``else:`` branch depending on the output of the condition.

Reviewer: ```suggestion
executes the ``then:`` branch if the condition returns true or the ``else:`` branch if the condition returns false.
```
